cypress/support/pages/tasks/pageCommonTasks.js

```diff
@@ -4,6 +4,13 @@ class Tasks_Common {
   static #tasksOverviewNavigationButton = '[data-testid="Aufgaben"]'
   static #taskForm = '[id="homework-form"]'
   static #submitButton = '[data-testid="submit-task-btn"]'
+  static #addTaskButton = '[data-testid="addTask"]'
+  static #taskNameInput = '[data-testid="homework-name"]'
+  static #homeWorkDescriptionP = '[class="ck ck-editor__main"]'
```

Comment:
Is this selector have some typo mistake or its intentional `#homeWorkDescriptionP`? Also its should be something like this `#homeworkDescriptionP`

Reply:
changed it to #homeworkDescription